arq/jobs.py

```diff
@@ -119,16 +126,30 @@ def __repr__(self):
         return f'<arq job {self.job_id}>'
 
 
-class PickleError(RuntimeError):
+class SerializationError(RuntimeError):
     pass
 
 
-def pickle_job(function_name: str, args: tuple, kwargs: dict, job_try: int, enqueue_time_ms: int):
+class PickleError(SerializationError):
+    pass
+
+
+def pickle_job(
```

Comment:
let's rename `unpickle_job` to `deserialize_job`, better to get it right now.

Change should be pretty explicit and easy to fix.